src/pages/en/reference/directives-reference.md

```diff
@@ -67,6 +67,25 @@ const cmsContent = await fetchHTMLFromMyCMS();
 <Fragment set:html={cmsContent}>
 ```
 
+`set:html={Promise<string>}` injects an HTML string into an element that is wrapped in a Promise.
+
+This can be used to inject HTML stored externally, such as a database.
```

Comment:
```suggestion
This can be used to inject HTML stored externally, such as in a database.
```
I think the HTML would be *in* a database, not the database itself?